fiatconnect-api.md

```diff
@@ -2569,6 +2570,29 @@ schema.
 
 The `url` field contains a URL which the user can follow in order to complete thier transfer of fiat funds to the provider.
 
+#### 9.3.3.5. `AccountNumberUserAction`
+
+`AccountNumberUserAction` is a User Action Details Schema for transfers in which require the user to send funds to 
+an account identified by some number to complete the transfer of fiat funds. 
+
+```
+{
+	userActionType: `TransferInUserActionDetailsEnum.AccountNumberUserAction`,
+	institutionName: `string`,
+	accountName: `string`,
+	accountNumber: `string`,
+	transactionReference: `string`,
```

Comment:
Can this be made optional for providers that don't require this? I imagine many providers will be able to cross reference the account the payment is from with the one on file. You can add the provision that if it's present, it MUST be included in the actual transfer itself.

Reply:
sounds reasonable. updated